src/main/java/com/ipiecoles/java/java220/Employe.java

```diff
@@ -1,7 +1,144 @@
 package com.ipiecoles.java.java220;
 
-/**
- * Created by pjvilloud on 21/09/17.
- */
-public class Employe {
+import org.joda.time.LocalDate;
+
+import java.util.Objects;
+
+public abstract class Employe {
+
+    public static String derniermatricule;
+
+    private String nom;
+    private String prenom;
+    private String matricule;
+    private LocalDate dateEmbauche;
+    private Double salaire;
+    private Boolean tempsPartiel;
+    private String sexe;
+
+    //constructeur
+    public Employe() {
+
+    }
+
+    public Employe(String nom, String prenom, String matricule, LocalDate dateEmbauche, Double salaire) {
+        this.nom = nom;
+        this.prenom = prenom;
+        this.matricule = matricule;
+        this.dateEmbauche = dateEmbauche;
+        this.salaire = salaire;
+    }
+
+    public Employe(String nom, String prenom, String matricule, LocalDate dateEmbauche, Double salaire, Boolean tempsPartiel, String sexe) {
+        this.nom = nom;
+        this.prenom = prenom;
+        this.matricule = matricule;
+        this.dateEmbauche = dateEmbauche;
+        this.salaire = salaire;
```

Comment:
Appelle le constructeur qui s'occupe déjà de l'instanciation des autres attributs
```suggestion
        this(nom, prenom, matricule, dateEmbauche, salaire);
```